data/campaigns/Eastern_Invasion/scenarios/99_Empire.cfg

```diff
@@ -0,0 +1,1514 @@
+        [/unit]
+
+        {GENERIC_UNIT 3 (Drake Clasher)      40 22} {FACING nw}
+        [+unit]
+            hitpoints=7
+        [/unit]
+
+        {GENERIC_UNIT 3 (Drake Glider)       40 23} {FACING nw}
+        {GENERIC_UNIT 3 (Drake Clasher)      43 23} {FACING nw}
+
+        # southwest units
+        {GENERIC_UNIT 1 (Orcish Archer)      26 20} {FACING sw}
+        [+unit]
+            hitpoints=19
+        [/unit]
+
+        {GENERIC_UNIT 4 (Drake Fighter)      25 21} {FACING ne}
+        [+unit]
+            hitpoints=3
+        [/unit]
+
+        {GENERIC_UNIT 4 (Drake Glider)       28 23} {FACING nw}
+        [+unit]
+            hitpoints=31
+        [/unit]
+
+        {GENERIC_UNIT 4 (Drake Fighter)      27 24} {FACING nw}
+
+        {GENERIC_UNIT 4 (Drake Fighter)      17 23} {FACING ne}
+        {GENERIC_UNIT 4 (Drake Burner)       18 22} {FACING ne}
+        {GENERIC_UNIT 4 (Drake Fighter)      18 23} {FACING ne}
+
+        # capture villages that start with a unit on them
+        [capture_village]
+            side=2
+            [filter]
+                side=2
+            [/filter]
+        [/capture_village]
+        [capture_village]
+            side=3
+            [filter]
+                side=3
+            [/filter]
+        [/capture_village]
+        [capture_village]
+            side=4
+            [filter]
+                side=4
+            [/filter]
+        [/capture_village]
+    [/event]
+
+    [event]
+        name=start
+        [scroll_to]
+            x,y=30,15
+            immediate=yes
+        [/scroll_to]
+
+        {MOVE_UNIT (id=Chief Dra-Nak) 28 18}
+        [message]
+            speaker=Chief Dra-Nak
+            message= _ "Filthy..."
+        [/message]
+        {MOVE_UNIT (id=Chief Dra-Nak) 29 17}
+        [message]
+            speaker=Chief Dra-Nak
+            message= _ "Stinking..."
+        [/message]
+        {MOVE_UNIT (id=Chief Dra-Nak) 30 15}
+        [message]
+            speaker=Chief Dra-Nak
+            message= _ "TRAITORS!!!"
+        [/message]
+        {MODIFY_UNIT (id=Chief Dra-Nak) facing se}
+        [message]
+            speaker=Chief Dra-Nak
+            message= _ "Drakes, human, dwarves... vermin, all of them! I offered them a chance at peace, at purpose! A chance to be part of something greater than themselves - my great golden empire!!! And this is the thanks I get?"
+        [/message]
+
+        {MOVE_UNIT (type=Orcish Slayer) 32 17}
+        [message]
+            type=Orcish Slayer
+            message= _ "Chief, more wyrms are attacking the east tunnels! There’s too many of them! What do we do?"
+        [/message]
+        [message]
+            speaker=Chief Dra-Nak
+            message= _ "I’m not going to let everything I’ve worked for fall apart so easily! These vermin are WEAK! We are STRONG!"
+        [/message]
+        [message]
+            type=Orcish Slayer
+            message= _ "Ok, but that’s not really a battle pla-"
```

Comment:
'Okay'

Reply:
I'd replace it with something entirely different, as "okay" feels overly modern to me. Maybe "fine", or "alright", or "very well", or something like that? I'm not sure of the context though so maybe something else is even better.

Reply:
Good point as well, my point was that if the text was supposed to read 'okay', it should be spelled as such.